test/e2e/cni/testdata/calico/mirror-calico.sh

```diff
@@ -15,30 +15,21 @@
 
 set -e
 
-CALICO_VERSION=$1
+SCRIPT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/" && pwd -P)"
 
-DST_REGISTRY="381492195191.dkr.ecr.us-west-2.amazonaws.com"
+CALICO_VERSION="${1:-$(cat "${SCRIPT_ROOT}/VERSION")}"
 
-if ! command -v oras &> /dev/null; then
-    echo "Please install oras"
-    exit 1
-fi
+ECR_ACCOUNT_ID="381492195191"
 
-# We use oras instead of the more typical docker pull/push to make
-# sure we mirror all architectures and digets is preserved.
-aws ecr get-login-password --region us-west-2 | oras login --username AWS --password-stdin ${DST_REGISTRY}
+for region in "us-west-2" "us-west-1"; do
+	registry="${ECR_ACCOUNT_ID}.dkr.ecr.${region}.amazonaws.com"
 
-for image in $(curl -s -L https://github.com/projectcalico/calico/releases/download/$CALICO_VERSION/metadata.yaml | yq  -r ".images[]" ); do
-	repo_with_version=$image
-	if [[ $image == quay* ]]; then
-		repo_with_version=${image#"quay.io/"}
-	else
-		image="docker.io/$image"
-	fi
-	repo=${repo_with_version%:*}
-	
-	aws ecr create-repository --repository-name $repo --region us-west-2 || true
-	
-	dst=${DST_REGISTRY}/${repo_with_version}
-	oras cp ${image} ${dst}
-done
+	for image in $(curl -s -L https://github.com/projectcalico/calico/releases/download/$CALICO_VERSION/metadata.yaml | yq  -r ".images[]" ); do
+		repo_with_version=$image
+		if [[ $image != quay* ]]; then
+			repo_with_version="docker.io/$image"
+		fi
+		
+		docker buildx imagetools inspect "${registry}/${repo_with_version}"
```

Comment:
could we add a comment here explaining that this does a head (or probably a get of the manifest?), which triggers the pull through in the ecr repo to populate the cache? If I didn't have context about this I think it wouldn't be obvious to me

same in the cilium script if you can